src/tests/ftest/container/boundary.py

```diff
@@ -0,0 +1,143 @@
+                self.log.info(
+                    "===(3.{0}.{1})Wrote {2} bytes to container {3}".format(pool_num, cont_num,
+                    data_bytes, container))
+            except (DaosTestError, TestFail) as err:
+                self.fail("#(3.{0}.{1}) container IO failed, err: {2}".format(
+                    pool_num, cont_num, err))
+        time.sleep(2)  #to syncup containers before close
+
+        try:
+            self.log.info("===(4.{0}.{1})create_container_and_test, container closing.".format(
+                pool_num, cont_num))
+            container.close()
+            self.log.info("===(4.{0}.{1})create_container_and_test, container closed.".format(
+                pool_num, cont_num))
+        except (DaosTestError, TestFail) as err:
+            self.fail("#(4.{0}.{1}) container close fail, err: {2}".format(
+                pool_num, cont_num, err))
+
+    def create_containers(self, pool=None, pool_num=10, num_containers=100, with_io=False):
+        """To create number of containers parallelly on pool.
+        Args:
+            pool(str): pool handle.
+            pool_num (int): pool number to create containers.
+            num_containers (int): number of containers to create.
+            with_io (bool): enable container test with execute_io.
+        """
+
+        self.log.info("==(2.{0})create_containers start.".format(pool_num))
+        thread_manager = ThreadManager(self.create_container_and_test, self.timeout - 30)
+
+        for cont_num in range(num_containers):
+            thread_manager.add(
+                pool=pool, pool_num=pool_num, cont_num=cont_num, with_io=with_io)
+        self.log.info("==(2.{0}) {1} containers created.".format(pool_num, num_containers))
+
+        # Launch the create_container_and_test threads
+        self.log.info("==Launching %d create_container_and_test threads", thread_manager.qty)
+        failed_thread_count = thread_manager.check_run()
+        if failed_thread_count > 0:
+            msg = "#(2.{0}) FAILED create_container_and_test Threads".format(failed_thread_count)
+            self.d_log.error(msg)
+            self.fail(msg)
+
+    def create_pools(self, num_pools=10, num_containers=100, with_io=False):
+        """To create number of pools and containers parallelly.
+        Args:
+            num_pools (int): number of pools to create.
+            num_containers (int): number of containers to create.
+            with_io (bool): enable container test with execute_io.
+        """
+
+        # Setup the thread manager
+        thread_manager = ThreadManager(self.create_containers, self.timeout - 30)
+
+        for pool_number in range(num_pools):
+            pool = self.get_pool()
+            thread_manager.add(
+                pool=pool, pool_num=pool_number, num_containers=num_containers, with_io=with_io)
+            self.log.info("=(1.{0}) pool created, {1}.".format(pool_number, pool))
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions